File: src/main/java/org/polyfrost/vanillahud/hooks/TooltipHook.java
package org.polyfrost.vanillahud.hooks;

import org.polyfrost.oneconfig.gui.animations.DummyAnimation;
import org.polyfrost.polyui.animate.Animation;

public class TooltipHook {

    public static boolean isScrolling;

    public static int gui$scrollY;

    public static Animation gui$animationY;

    public static void resetScrolling() {
        gui$scrollY = 0;
    }

}

File: src/main/kotlin/org/polyfrost/vanillahud/hooks/TooltipHook.kt
package org.polyfrost.vanillahud.hooks

import org.polyfrost.polyui.animate.Animation

object TooltipHook {
    @JvmField
    var isScrolling: Boolean = false

    @JvmField
    var scrollY: Int = 0

    @JvmField
    var animationY: Animation? = null

    @JvmStatic
    fun resetScrolling() {
        scrollY = 0
    }
}